raiden/tests/integration/network/transport/test_udp.py

```diff
@@ -32,7 +32,7 @@ def test_udp_reachable_node(raiden_network, skip_if_not_udp):
 
 
 @pytest.mark.parametrize('number_of_nodes', [2])
-def test_udp_unreachable_node(raiden_network, skip_if_not_udp):
+def test_udp_unreachable_node(raiden_network, skip_if_not_udp):  # pylint: disable=unused-argument
```

Comment:
I guess all those "marker" fixture arguments should eventually be moved to `@pytest.mark.parametrize()` statements.